### ISSUE TITLE
Deaccessioned studies included in metadata export

### ISSUE THREAD

**Issue by johnhuck** [ualbertalib/dvn#34]
@piyapongch , could you please investigate why the following de-accessioned studies were included in the most recent metadata export. When a study is deaccessioned, there should be no released version of the study. My presumption is that only released studies are exported. Is it possible to explain why it happened in this case? Thank you.

http://dx.doi.org/10.7939/DVN/10314
http://dx.doi.org/10.7939/DVN/10470

(Note that 10470 also belongs to an unreleased dataverse).

**Comment by piyapongch** [comment]
I will take a look at it.

**Comment by johnhuck** [comment]
Hi Piyapong,

After our conversation today where you suggested I check the date last modified for the two de-accessioned items I did check and found that they are earlier dates. I looked at a number of other files and they all had modified dates of September 30th.

The DDI XML file for 10314 was last modified May 1, 2015.
The DDI XML file for 10470 was last modified June 9, 2015.

Study 10315 was released: Thu Apr 30 09:37:59 MDT 2015 – Archived: Sun Jun 21 13:10:09 MDT 2015
Study 10470 was released: Mon Jun 08 14:06:44 MDT 2015 – Archived: Thu Jun 11 12:54:05 MDT 2015

So this appears to confirm your hypothesis that the Dataverse export process may be writing over metadata files in the directory it exports to that were previously exported, and that since there would be no new file with which to write over an existing file for a de-accessioned study, the existing file would remain in the directory.

The next time we try a metadata export, we can ask Henry to wipe the export directory clean first. If we don't find these two studies in the export output, then that would also confirm the hypothesis.
